### PR TITLE
SDXL: Fix mslite infer bug

### PR DESCRIPTION
**Pull request by zhanghuiyao**

### CODE REVIEW

File: examples/stable_diffusion_xl/inference/README.md [hunk 1]
```diff
@@ -33,13 +30,10 @@ Refer to [Lite install](https://mindspore.cn/lite/docs/zh-CN/r2.1/use/downloads.
...
    ```
-**Note: MindSpore and MindSpore Lite must be same version.**
 
-</details>
-
-## Pretrained Weights
+## Preparation
 
-### Convert Pretrained Checkpoint
+### First, Convert Pretrained Checkpoint to `.ckpt` format
```
Comment (CaitinZhao) [inline examples/stable_diffusion_xl/inference/README.md:36]: 这块标题重新理下吧

## install
## Inference
### Preparation
### Offline Infer
#### 1. export
#### 2. convert
#### 3. infer
### Onine Infer
#### infer

**Review by SamitHuang — APPROVED** [review]
grammar errors are to be fixed in readme (e.g. via Grammarly)

File: examples/stable_diffusion_xl/inference/README.md (hunk 1 above)
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/inference/README.md]: done

File: examples/stable_diffusion_xl/inference/README.md [hunk 2]
```diff
@@ -2,22 +2,19 @@
...
 
-### Install MindSpore Lite
+### 2. Install MindSpore Lite
 
-Refer to [Lite install](https://mindspore.cn/lite/docs/zh-CN/r2.1/use/downloads.html)
+Refer to [Lite Install](https://mindspore.cn/lite/docs/zh-CN/r2.1/use/downloads.html)
 
 1. Download the supporting tar.gz and whl packages according to the environment.
-2. Unzip the tar.gz package and install the corresponding version of the WHL package.
+2. Unzip the tar.gz package and install t corresponding version of the WHL package.
```
[the remark this answers is not in the document]
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/inference/README.md]: done

**Comment by zhanghuiyao** [comment]
> grammar errors are to be fixed in readme (e.g. via Grammarly)

done

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**